src/controllers/orderController.js

```diff
@@ -23,6 +31,22 @@ async function getAllOrders (req, res) {
 async function addOrder (req, res) {
 
     //implemant the code here
+    const { dish_id, customer_id, order_description } = req.body;
```

Comment:
you should check whether this customer_id and dish_id exist or not.

Reply:
is there any need to import customerSchema and dishSchema in my code?

Reply:
instead of using customer and dish schemas its better to call their api directly from here. look at my code addDish function in dishController, I used axios for calling Api, so you can have data, instead of calling schema which leads to accessing database too much and some problems.